.coveragerc

```diff
@@ -379,6 +379,8 @@ omit =
     homeassistant/components/plum_lightpad/*
     homeassistant/components/point/*
     homeassistant/components/prometheus/*
+    homeassistant/components/ps4/__init__.py
```

Comment:
The access mode has been changed on `.coveragerc`. Please revert that.

Reply:
Ok Should be right now. Don't know how that happened.